themes/cardinal_service/pdb_components/opportunities_list/src/Components/SelectList.jsx

```diff
@@ -0,0 +1,76 @@
+import React, {Component} from 'react';
+import Select from 'react-select';
+
+const _ = require('lodash-uuid');
+
+export class SelectList extends Component {
+
+  constructor(props) {
+    super(props);
+    this.uuid = _.uuid();
+    this.state = {selectedItems: []};
+    this.onChange = this.onChange.bind(this);
+  }
+
+  onChange(selectedOptions) {
```

Comment:
We should probably discuss and define our standards for JSX templates. I feel that each function probably should have a comment.